platform/view/services/storage/driver/sql/postgres/binding_test.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package postgres
+
+import (
+	"context"
+	"testing"
+
+	testing2 "github.com/hyperledger-labs/fabric-smart-client/platform/view/services/storage/driver/common/testing"
+
+	"github.com/hyperledger-labs/fabric-smart-client/platform/view/view"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPutBindingsMultipleEphemerals(t *testing.T) {
```

Comment:
```suggestion
func TestPutBindingsMultipleEphemeralsPostgres(t *testing.T) {
```

Let me suggest to add Postgres to the test name to indicate this test runs with Postgres. I am suggesting this as I am currently trying to "tidy" up the postgres tests a bit in #1069.